Makefile

```diff
@@ -95,16 +95,26 @@ hugo/run:
 run: docker/build
 	$(DOCKER_RUN) hugo/run
 
+## Build customized utterances widget
+utterances/build:
+	rm -rf utterances
+	git clone --branch master https://github.com/cloudposse/utterances.git
```

Comment:
Hrmm... this is shaddowing the `utterances` submodule

Reply:
https://github.com/cloudposse/docs/pull/180/files#diff-e168af1b6b9585501e72cb8ff038183b